Bottom: b75d7017886e54384b8a47cb07fd2ca0e7ccb603
Top:    f1130c4077b57c0a049f2b4da3624a10cf2eda0e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-27 13:51:31 +0100

Refresh of try-to-make-gdk-and-gl-version

---

diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index 9445075..d4e1746 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -911,6 +911,7 @@ ghid_show_crosshair (gboolean show)
     }
 }
 
+void
 ghid_init_renderer (int *argc, char ***argv, GHidPort *port)
 {
   /* Init any GC's required */
